library/Backends/Michelson/src/Juvix/Backends/Michelson/DSL/InstructionsEff.hs

```diff
@@ -58,6 +58,8 @@ inst (Types.Ann _usage ty t) =
   case t of
     Ann.Var symbol -> var symbol
     Ann.AppM fun a -> appM fun a
+    Ann.UnitM -> undefined
```

Comment:
this should probably not be undefined? or will this never get called? at least `error` if so?

Reply:
Sure I"ll change it to error, I was going to fix it in this pr anyways, since it's just a pair and unit call

Reply:
both converted, just do a quick look over at the pair and unit translations